data/coalition/coalition news.txt

```diff
@@ -431,3 +435,34 @@ news "coalition games crowd"
 			"The lines of people heading to the Coalition Games seem to have stopped moving abruptly. Upon further investigation, you see some young Kimek have fallen on their backs and are being helped back to their feet one by one by both Heliarch agents and others from the crowd."
 			"This is a particularly crowded part of the spaceport today. The lines of people waiting for admission into the Games have started crossing and looping in on themselves. Some even head inside shops and restaurants while others stretch into docking areas. Battalions of Heliarch agents work as hard as they can to bring everyone back into orderly lines."
 			"Several children coming to see the Coalition Games are running and playing simplified versions of the events, sporting different colors and assets for different games. Their parents struggle to get them to settle down while keeping their spots in the waiting lines."
+
+news "lunarium charity"
+	location
+		government "Coalition"
+		not planet "Fourth Shadow"
+	name
+		word
+			"Charity"
```

Comment:
Maybe add organization? Or charitable organization?

Reply:
Yeah, charitable organization is better I'd say. Still not great since these are usually people you're interacting with, but since this is a scene rather than a quote, I guess there's not much we can improve on.
```suggestion
			"Charitable Organization"
```